frontend/lib/src/controllers/auth_header_controller.dart

```diff
@@ -15,4 +18,10 @@ class AuthHeaderController extends GetxController {
 
     return name;
   }
+
+  Future<bool> getLoginToken() async {
```

Comment:
rename to something else, its not getting the login token, 
previous prop `hasAuthToken`  is ok or hasLoginToken